eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -1,6 +1,6 @@
 cabal-version: 3.0
 name:          cardano-ledger-alonzo-test
-version:       1.1.2.8
+version:       1.2
```

It is only the cabal file that changed in this package. Patch bump
```suggestion
version:       1.1.2.9
```